sdks/python/apache_beam/io/gcp/gcsio.py

```diff
@@ -99,6 +99,40 @@ def get_or_create_default_gcs_bucket(options):
         bucket_name, project, location=region)
 
 
+def create_storage_client(pipeline_options, use_credentials=True):
+  """Create a GCS client for Beam via GCS Client Library.
+
+  Args:
+    pipeline_options(apache_beam.options.pipeline_options.PipelineOptions):
+      the options of the pipeline
+    use_credentials(bool): whether to retrieve credentials from pipeline
+      options or not
```

Comment:
Lets clarify what happens if use_credentials=False. For example:

whether to create an authenticated client based on pipeline options or an anonymous client.

nit: `.` at the end of sentences in the docstring.